pupil-api/src/App.ts

```diff
@@ -12,6 +12,8 @@ import * as winston from 'winston'
 import * as expressWinston from 'express-winston'
 import * as azure from './azure'
 const config = require('./config')
+const corsOptions = require('../../admin/helpers/cors-options')
+const setupLogging = require('../../admin/helpers/logger')
```

Comment:
Same comment as above here.

Reply:
agree on this too.  please include in this project and also allow for setup via env vars.  IIRC thats how the admin app does it.